### ISSUE TITLE
Stop on Full Inventory

### ISSUE THREAD

**Issue by alezzar** [kolton/d2bot-with-kolbot#1235]
Hi guys,

How do you make the bot stop when inventory is full?

Thanks in advance!

**Comment by drogga** [comment]
There was something on projectetal but the site is gone
you have to edit the code for the Bot to stop on full Inventory
Better get a second key and setup AutoMule :)

**Comment by alezzar** [comment]
I'm currently running on 2 different cd keys, which mean I'd need 3rd one :D

**Comment by DarkHorseDre** [comment]
> I'm currently running on 2 different cd keys, which mean I'd need 3rd one :D

no, automule has a setting to stop a specified profile to use the key for the mule.  you could mule one then the other.
